docs/source/platform_technical_documentation/custom-environments.ipynb

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
<p>Maybe just call this section Workspaces? We should probably raise a ticket that it's unclear that he Basic Stable is <code>python</code> and that all other workspaces also include <code>python</code> as part of <code>conda</code> environments.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Changed the title. </p><p> </p><blockquote>We should probably raise a ticket that it's unclear that he Basic Stable is <code>python</code> and that all other workspaces also include <code>python</code> as part of <code>conda</code> environments.</blockquote><p>You mean that the workspace titles do not reflect what they contain ? Agreed. </p>

Reply:
created https://github.com/MAAP-Project/maap-documentation/issues/163